[2023-01-04T13:42:46.625+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_pipeline.Postgres_Populating_Database scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-01-04T13:42:46.637+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_pipeline.Postgres_Populating_Database scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-01-04T13:42:46.637+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T13:42:46.638+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-01-04T13:42:46.639+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T13:42:46.657+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): Postgres_Populating_Database> on 2023-01-02 00:00:00+00:00
[2023-01-04T13:42:46.664+0000] {standard_task_runner.py:55} INFO - Started process 650 to run task
[2023-01-04T13:42:46.668+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'accidents_pipeline', 'Postgres_Populating_Database', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/Airflow.py', '--cfg-path', '/tmp/tmp4hhoyax9']
[2023-01-04T13:42:46.670+0000] {standard_task_runner.py:83} INFO - Job 44: Subtask Postgres_Populating_Database
[2023-01-04T13:42:46.746+0000] {task_command.py:389} INFO - Running <TaskInstance: accidents_pipeline.Postgres_Populating_Database scheduled__2023-01-02T00:00:00+00:00 [running]> on host 980a01aa00ba
[2023-01-04T13:42:46.827+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=accidents_pipeline
AIRFLOW_CTX_TASK_ID=Postgres_Populating_Database
AIRFLOW_CTX_EXECUTION_DATE=2023-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-02T00:00:00+00:00
[2023-01-04T13:42:47.559+0000] {logging_mixin.py:137} INFO - connected succesfully
[2023-01-04T13:43:30.000+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-04T13:43:30.009+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=accidents_pipeline, task_id=Postgres_Populating_Database, execution_date=20230102T000000, start_date=20230104T134246, end_date=20230104T134330
[2023-01-04T13:43:30.092+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-04T13:43:30.115+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
